src/components/atoms/IngredientMenu/IngredientMenu.tsx

```diff
@@ -0,0 +1,43 @@
+import { Menu } from "@mantine/core"
+import Image from "next/image"
+
+import { ChevronDown, ChevronUp, Pencil, Trash } from "tabler-icons-react"
+
+interface IngredientMenuProps {
+  showMoveUp: boolean
+  showMoveDown: boolean
+  showEdit: boolean
+}
+
+export const IngredientMenu = ({ showMoveUp = true, showMoveDown = true, showEdit = true }: IngredientMenuProps) => {
```

Comment:
必須で受け取るpropsの初期値設定は不要かと思います 👍 
```suggestion
export const IngredientMenu = ({ showMoveUp, showMoveDown, showEdit }: IngredientMenuProps) => {
```

Reply:
消しました！